### DISCUSSION TITLE
无法 push 更新到 Github Repo

### DISCUSSION THREAD

**Discussion by botsp** [rbind/yihui#1530] [Q&A] [answered]
感谢分享的主题!
我现在使用R studio cloud已经成功链接到Gitub_repo，但是现在无法push更新到repo，可以帮忙调查下原因吗，是否是cloud端导致的问题呢
（不知道往哪个方向去解决这个问题~
![image](https://github.com/yihui/yihui.org/assets/147964952/e627a37c-a271-4b97-9a18-8a4c609c3bba)

**Comment by botsp** [comment]
pull from Github是可以的

**Reply by chuxinyuan** [reply, marked as answer]
先点击 Commit 按钮，然后再点击 Push 按钮。

**Reply by chuxinyuan** [reply]
另外，你可以参考下 [Happy Git and GitHub for the useR](https://happygitwithr.com/)

**Reply by botsp** [reply]
感谢，解决了！
